scripts/apidoc_full.sh

```diff
@@ -12,17 +12,37 @@
 
 set -eu
 
-CONFIG_H='include/mbedtls/mbedtls_config.h'
+. $(dirname "$0")/project_detection.sh
 
-if [ -r $CONFIG_H ]; then :; else
-    echo "$CONFIG_H not found" >&2
+CRYPTO_CONFIG_H='include/psa/crypto_config.h'
+if in_mbedtls_repo; then
+    CONFIG_H='include/mbedtls/mbedtls_config.h'
+    CRYPTO_CONFIG_H='/tf-psa-crypto/include/psa/crypto_config.h'
+
+    if [ -r $CONFIG_H ]; then :; else
+        echo "$CONFIG_H not found" >&2
+        exit 1
+    fi
+fi
+
+if [ -r $CRYPTO_CONFIG_H ]; then :; else
+    echo "$CRYPTO_CONFIG_H not found" >&2
     exit 1
 fi
 
-CONFIG_BAK=${CONFIG_H}.bak
-cp -p $CONFIG_H $CONFIG_BAK
+CRYPTO_CONFIG_BAK=${CRYPTO_CONFIG_H}.bak
+cp -p $CRYPTO_CONFIG_H $CRYPTO_CONFIG_BAK
 
-scripts/config.py realfull
-make apidoc
+if in_mbedtls_repo; then
+    CONFIG_BAK=${CONFIG_H}.bak
+    cp -p $CONFIG_H $CONFIG_BAK
+    scripts/config.py realfull
+    make apidoc
+    mv $CONFIG_BAK $CONFIG_H
+elif in_tf_psa_crypto_repo; then
+    scripts/config.py full
```

Comment:
```suggestion
    scripts/config.py realfull
```
and you need to add support for realfull in config.py in TF-PSA-Crypto as done in config.py in mbedtls. This should fix the "documentation for unknown define MBEDTLS_xyz found." warnings.